Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2428,6 +2435,58 @@ def to_context(self):
             }
             return ret_value
 
+    class CustomIndicator(Indicator):
+
+        def __init__(self, indicator_name, value, dbot_score, params, prefix_str):
+            """
+            :type indicator_name: ``Str``
+            :param indicator_name: name of the indicator
+            :type value: ``Any``
+            :param value: Value of the indicator
+            :type dbot_score: ``DBotScore``
+            :param dbot_score: If custom indicator has a score then create and set a DBotScore object.
+            :type params: ``Dict(Str,Any)``
+            :param params: A dictionary containing all the param names and their values
+            :type prefix_str: ``Str``
+            :param prefix_str: Will be used as the context path prefix
+            :return: None
+            :rtype: ``None``
```

Comment:
In the original issue, created by Anar, he said that we should create an indicator which will let users pass a dictionary that will contain the parameters wanted for this indicator. Because it's custom the params need to be passed by the user of this class